client/src/elm/Pages/MessagingCenter/View.elm

```diff
@@ -427,40 +428,50 @@ viewTabs language model =
 viewResilienceMessage : Language -> NurseId -> Nurse -> Model -> ( ResilienceMessageId, ResilienceMessage ) -> Html Msg
 viewResilienceMessage language nurseId nurse model ( messageId, message ) =
     let
-        ( extraClass, ( head, body ) ) =
+        messageCategory =
+            span [ class "category-header" ] [ text <| translate language <| Translate.ResilienceCategory message.category ]
+
+        ( extraClass, header, ( head, body ) ) =
```

Comment:
If `header` is `messageCategory` for all, then we don't need to resolve it according to `message.category`.
Make it `( extraClass, ( head, body ) ) = ` ...